INSTALLATION.md

```
@@ -13,44 +71,535 @@
+   - Execute the following command in your terminal:
+     ```bash
+     cp ./envFiles/.env.production ./.env
+     ```
+
+Choose whichever method suits you best!
+
+### Step 4: Configuring Environment Variables
+
+Populate the following variables in your `.env` file:
+
+    1. **CADDY_TALAWA_API_DOMAIN_NAME** (e.g., `localhost`)
```

Markdownlint (0.37.0)

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)